sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/ResourceExtensions.cs

```diff
@@ -119,7 +119,20 @@ internal static class ResourceExtensions
             }
         }
 
-        if (metricsData != null)
+        bool shouldReportMetricTelemetry = false;
+        try
+        {
+            var exportResource = Environment.GetEnvironmentVariable("EXPORT_METRIC_FOR_RESOURCE_ATTRIBUTES");
```

Comment:
how about `ENABLE_AZURE_MONITOR_OTEL_RESOURCE_METRICS`?

Reply:
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/configuration/sdk-environment-variables.md#language-specific-environment-variables

OTEL_DOTNET_AZURE_MONITOR_ENABLE_RESOURCE_METRICS 
Should we use the otel guidance?

Reply:
Please create a const for the environment variable in https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/EnvironmentVariableConstants.cs

Reply:
Done